Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.2.1"
 description: A PowerDNS web interface with advanced features.
 name: powerdnsadmin
-version: 0.2.0
+version: 0.2.1
```

```suggestion
version: 0.3.0
```

Its not a patch release, cause its has breaking configs.